tests/bluemira/equilibria/test_st_equilibrium.py

```diff
@@ -180,22 +180,54 @@ def test_equilibrium(self):
             convergence=criterion,
         )
         fbe_iterator()
-        self._test_equilibrium_good(eq, psi_rtol=1e-1, li_rtol=1e-2)
+        self.eq = eq
+        self._test_equilibrium_good(eq)
+        self._test_profiles_good(eq)
 
         # Verify by removing symmetry constraint and checking convergence
         eq.force_symmetry = False
         eq.set_grid(grid)
         fbe_iterator()
         # I probably exported the eq before it was regridded without symmetry..
-        self._test_equilibrium_good(eq, psi_rtol=1e-1, li_rtol=1e-3)
+        self._test_equilibrium_good(eq)
 
-    def _test_equilibrium_good(self, eq, psi_rtol, li_rtol):
-        lcfs_area = eq.get_LCFS().area
-        assert np.isclose(self.eq_blueprint.get_LCFS().area, lcfs_area, rtol=1e-3)
+        self._test_profiles_good(eq)
 
-        li_bp = calc_li(self.eq_blueprint)
-        assert np.isclose(li_bp, calc_li(eq), rtol=li_rtol)
-        assert np.allclose(self.eq_blueprint.psi(), eq.psi(), rtol=psi_rtol)
+    def _test_equilibrium_good(self, eq):
+        assert np.isclose(eq._Ip, abs(self.jeq_dict["cplasma"]))
+        lcfs = eq.get_LCFS()
+        assert np.isclose(self.eq_blueprint.get_LCFS().area, lcfs.area, rtol=1e-2)
+        assert np.isclose(lcfs.centroid[-1], 0.0)
+
+    def _test_profiles_good(self, eq):
+        """
+        Test the profiles are the same shape. Normalisation won't be one: JETTO is
+        fixed boundary and has a different plasma volume.
+        """
+
+        def scale(profile):
+            return np.abs(profile) / np.max(np.abs(profile))
+
+        jetto_pprime = self.jeq_dict["pprime"]
+        jetto_ffprime = self.jeq_dict["ffprime"]
+
+        psi_n = self.jeq_dict["psinorm"]
+        bm_pprime_p = self.profiles.pprime(psi_n)
+        bm_ffprime_p = self.profiles.ffprime(psi_n)
+
+        bm_pprime = eq._profiles.pprime(psi_n)
+        bm_ffprime = eq._profiles.ffprime(psi_n)
+        assert np.allclose(bm_pprime, bm_pprime_p)
+        assert np.allclose(bm_ffprime, bm_ffprime_p)
+        assert np.isclose(max(bm_ffprime) / max(jetto_ffprime), abs(eq._profiles.scale))
+        assert np.isclose(max(bm_pprime) / max(jetto_pprime), abs(eq._profiles.scale))
```

Comment:
Minor suggestion - could add similar tests for pressure and fRBpol here too, now they're being scaled